Parsers/ASimAuthentication/Parsers/ASimAuthenticationVectraCloudAudit.yaml

```diff
@@ -0,0 +1,54 @@
+Parser:
+  Title: Authentication ASIM parser for VectraCloud Audit Logs Event
+  Version: '0.1'
+  LastUpdated: Apr 18 2023
+Product:
+  Name: Vectra
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing VectraCloud Audit Logs Event in the Audits_Data_CL table to the ASIM Authentication schema.
+ParserName: ASimAuthenticationVectraCloudAudit
+EquivalentBuiltInParser: _ASim_Authentication_VectraCloudAudit
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled:bool = false)
+  {
+    Audits_Data_CL
+    | where event_action_s in ("login","logout")
+    | extend
+          EventCount = int(1),
+          EventEndTime = event_timestamp_t,
+          EventProduct = 'Vectra Cloud',
+          EventResult = case(result_status_s=="success", "Success", result_status_s=="failure", "Failure","NA"),
+          EventSchema = "Authentication",
+          EventSchemaVersion = "0.1.3",
+          EventStartTime = event_timestamp_t,
+          EventType = case(event_action_s=="login", "Logon", event_action_s=="logout", "Logoff","Elevate"),
+          EventVendor = 'Vectra',
+          ActorUserId = tostring(toint(user_id_d)),
+          ActorUserIdType = "VectraUserId",
+          ActorUsernameType = "UPN",
+          EventUid = tostring(toint(id_d))
+    | project-rename
+          DvcIpAddr = source_ip_s,
```

Comment:
Is this IP of the source user or the device creating this event?

Reply:
@devikamehra The source_ip_s will contain the IP address of the device from which the event is generated.